commands/trigger_job.go

```diff
@@ -30,11 +36,19 @@ func (command *TriggerJobCommand) Execute(args []string) error {
 		return err
 	}
 
-	build, err := target.Team().CreateJobBuild(pipelineName, jobName)
-	if err != nil {
-		return err
+	if &buildNumber != nil {
```

Comment:
This will never be false, because `buildNumber` is definitely allocated, so its pointer can never be nil.  I think you want something more like `if buildNumber != 0`.